### PR TITLE
Add tasklib recipe

### PR DESCRIPTION
**Pull request by jfishe**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by jfishe** [comment]
@conda-forge-admin, please ping team @conda-forge/help-python tasklib is ready for review.

File: recipes/tasklib/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/tasklib-{{ version }}.tar.gz
+  sha256: 5ccd731b52636dd10457a8b8d858cb0d026ffaab1e3e751baf791bf803e37d7b
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
```
Comment (beenje) [inline recipes/tasklib/meta.yaml:14]: This should be a `noarch: python` recipe.

File: recipes/tasklib/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - backports.zoneinfo  # [py<39]
```
Comment (beenje) [inline recipes/tasklib/meta.yaml:22]: You can't use selector with `noarch` recipe but you don't have to.
The `backports.zoneinfo` conda-forge package won't install anything for python >= 3.9. It's one of those [empty packages](https://conda-forge.org/docs/maintainer/knowledge_base.html#empty-python-packages) in that case. You can put it as run requirement for all python versions.

File: recipes/tasklib/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+  imports:
+    - tasklib
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/GothenburgBitFactory/tasklib
+  summary: Official Taskwarrior library for Python
```
Comment (beenje) [inline recipes/tasklib/meta.yaml:34]: You could add: `doc_url: https://tasklib.readthedocs.io/`

**Review by beenje — CHANGES_REQUESTED** [review]

**Pull request merged**